frontend/src/views/invite/InviteView.vue

```diff
@@ -0,0 +1,133 @@
+<script setup lang="ts">
+import { storeToRefs } from 'pinia';
+import { ref, onMounted } from 'vue';
+import { useRouter } from 'vue-router';
+
+import { Spinner } from '@/components/layout';
+import { Button, Message, useToast } from '@/lib/primevue';
+import { inviteService } from '@/services';
+import { useConfigStore } from '@/store';
+import { RouteNames } from '@/utils/constants';
+
+import type { Ref } from 'vue';
+
+// Types
+interface ErrorType {
+  [key: string]: {
+    title: string;
+    detail: string;
+  };
+}
+
+// Props
+type Props = {
+  token: string;
+};
+
+const props = withDefaults(defineProps<Props>(), {
+  token: undefined
+});
+
+const router = useRouter();
+
+// Store
+const { getConfig } = storeToRefs(useConfigStore());
+
+// State
+const invalidInvite: Ref<boolean> = ref(false);
+const title: Ref<string> = ref('Invalid Invite');
+const detail: Ref<string> = ref('Please check your link again');
+
+// Actions
+const toast = useToast();
+
+const errorMessage: ErrorType = {
+  401: {
+    title: 'Unauthorized',
+    detail: 'Invalid authorization credentials'
+  },
+  403: {
+    // wrong email
+    title: 'Forbidden',
+    detail: 'The person who sent you the link may have restricted it to another email address'
+  },
+  404: {
+    // invitation not found/expired
+    title: 'Not found',
+    detail: 'Invitation not found or has expired'
+  },
```

Comment:
If the invitation is expired, COMS will report a 410. We might need to define a 410 key with the same contents as 404.